docs/examples/Space_Robot.rst

```diff
@@ -0,0 +1,538 @@
+=============================================
+Astrobee: A Holonomic Free-Flying Space Robot
+=============================================
```

Comment:
@abhikamath Both of these examples could use, at minimum, a paragraph introduction explaining what the model is, what it does, and any basic intro information.